README.md

```diff
@@ -0,0 +1,72 @@
+# JavaScript Capstone Restuarant APP using API
+
+> This is a Restuarant App Developed in Vanilla JavaScript. The main objective for this project is to learn about basic JavaScript concepts, functions, arrays and objects DOM manipulation and connect to API's.
+
+### Screenshots
+
+|Main Project page
+
+|![Fastantistic Restaurant](https://user-images.githubusercontent.com/30318155/129350424-7d1d5391-4d25-44d0-94a0-b08659b2d95a.png)|
+
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap
+- Font Awesome
+- JavaScript-
+- MDB API
+- Involvement API
+- Jest Test
+
+## My Project
+
+This project is found on the following Repo [Fantastic Restaurant](https://github.com/den-cloud/JS-CAPSTONE-PROJECT/)
+
+## Live Demo
```

Comment:
- [ ] The live of the project is broken,  the link takes you back to the your repo,  please kindly fix that.